tools/mo/unit_tests/mo/utils/mo_fallback_test_actual.py

```diff
@@ -291,6 +298,8 @@ def test_no_fallback_if_pdpd(self, use_new_fe, use_legacy, extension, expected_p
 
 
```

Comment:
I think we should mark such tests as skipped if pdpd is not imported

Reply:
Please use pytest marking and replace `generate` to pytest parameterize

Reply:
Done.